source/partials/_workshops.html.slim

```diff
@@ -2,28 +2,27 @@
   - data.workshops.workshop_list.each do |workshop|
     .c-workshop.col-1-2 id=workshop.slug
       .c-workshop__details
-        h3.c-workshop__title = workshop.title
-        .c-workshop__detail
-          p
-            span= "#{workshop.long}"
-            = "#{workshop.level}"
+        h2.c-workshop__title = workshop.title
+        h3.c-workshop__detail
+          = "#{workshop.long}"
+          span= "#{workshop.level}"
         .c-workshop__description
           p = workshop.description
 
       .c-workshop__presenter
         - workshop.presenters.each do |presenter|
           .c-workshop__presenter-details
-            img.c-speaker__avatar.c-speaker__avatar--small src=("/images/workshop-avatars/#{presenter.avatar}") alt=(presenter.name)
-            span.c-presenter__title = "#{presenter.name}"
+            img.c-speaker__avatar.c-speaker__avatar--small src=("/images/workshop-avatars/#{presenter.avatar}.jpeg") alt=(presenter.name)
```

Comment:
⚠️ This doesn't work
<img width="425" alt="screen shot 2017-08-06 at 11 56 53 pm" src="https://user-images.githubusercontent.com/1615322/29003008-f49df4c6-7b02-11e7-8725-aa0bda8b042b.png">

And I don't think it's a good idea - don't want to be restricted to using only `.jpeg`